### ISSUE TITLE
Storybook + Next + Netlify

### ISSUE THREAD

**Issue by tolicodes** [storybookjs/storybook#17031]
**Describe the bug**
I am getting an error when trying a storybook build using the Netlify next plugin

**To Reproduce**
Install a next app, then add storybook. 
Npm install `@netlify/plugin-nextjs`
After it's installed, set up a new site for storybook in Netlify:

build: `next build-storybook` 
output: `storybook-static`

```
2:33:01 PM:   Error: The directory "storybook-static" does not contain a Next.js production build. Perhaps the build command was not run, or you specified the wrong publish directory.
```

**System**
Please paste the results of `npx sb@next info` here.

**Comment by baby636** [comment]
https://issuehunt.io/r/baby636/Hackathons

**Comment by Terkea** [comment]
Same here

**Comment by tolicodes** [comment]
Yeah. The old plug-in was giving me another issue - it somehow uploaded the next cache to lambda (exceeding the 60MB limit). The actual packed app is only 2M

**Comment by alk-mandrianarijaona** [comment]
hi @tolicodes do you happen to have a reproduction repo for this issue?

**Comment by alk-mandrianarijaona** [comment]
@tolicodes I have created a reproduction repository https://github.com/alk-mandrianarijaona/fix-storybook-17031 but so far I can't reproduce the issue. Can you please share your `netlfiy.toml` file or your build configuration in netlify?

**Comment by valentinpalkovic** [comment]
This was a config issue between `@netlify/plugin-nextjs` and the (now-removed) `build-storybook` CLI. Storybook 7+ uses `storybook build`, and Netlify Next deployments don't wrap Storybook. Closing as stale (4.4y, 0 thumbs-up); please open a fresh issue with a v10 + current Netlify config if it still fails.
